jac-uk/assessments
# #1531 firebase app check

## What's included?
This change imports the firebase app check into the admin repository and works around an issue with the application not building on some machines or environments.

## Who should test?

✅ Developers


## How to test?
Add the public recaptcha key below into your .env.develop.local file and make some requests. Ensure that these show in the verified requests on the firebase project settings => app check in the firebase console.

`6LcVn5kbAAAAADZNrAZCfEJ2KOCyS--Alb7eHvXG`

## Risk - how likely is this to impact other areas?

🟠 Medium risk - this does change code that is shared with other areas *however* enforcement isn't enabled for form requests and firebase functions/firestore changes as yet.


## Additional context


---
PREVIEW:DEVELOP

This work can only be previewed on localhost as of now.
